openfisca_uk/variables/finance/tax/tax.py

```diff
@@ -9,8 +9,16 @@ class tax(Variable):
     definition_period = YEAR
 
     def formula(person, period, parameters):
-        return person("income_tax", period) + person(
-            "national_insurance", period
+        ct_payment = person("is_household_head", period) * person.household(
+            "council_tax", period
+        )
+        return (
+            add(
```

Comment:
Could be more readable as a separate `income_tax_ni` variable

Reply:
Yeah - this would involve a lot of changes/renames as `tax` is used quite a lot throughout.

Reply:
just mean python variable not openfisca variable